src/fits.jl

```diff
@@ -1,11 +1,13 @@
 # helper functions
 
-#=
-FITSIO.jl takes over memory read in by cfitsio, which reads in row-major form,
-whereas when Julia takes that memory, it is assumed as column major.
-Therefore all data read by `read` is transposed.
-Related comment: https://github.com/JuliaAstro/CCDReduction.jl/pull/16#issuecomment-638492572
-=#
+"""
+    getdata(::FITSIO.ImageHDU)
```

Comment:
Is this exported? If not, I would change to `CCDReduction.getdata(::FITSIO.ImageHDU)`